bigscience/gins/task.py

```diff
@@ -43,6 +43,134 @@ def full_lm(dataset, sequence_length, output_features):
+    "super_glue_copa_exercise",
+    "super_glue_copa_i_am_hesitating",
+    "super_glue_copa_more_likely",
+    "super_glue_copa_plausible_alternatives",
+    "super_glue_copa__As_a_result_C1_or_C2_",
+    "super_glue_copa__What_could_happen_next_C1_or_C2_",
+    "super_glue_copa__which_may_be_caused_by",
+    "super_glue_copa__why_C1_or_C2",
+
+    # HellaSwag
+    "hellaswag_Predict_ending_with_hint",
+    "hellaswag_Randomized_prompts_template",
+    "hellaswag_complete_first_then",
+    "hellaswag_if_begins_how_continues",
+
+    # Story Cloze
+    "story_cloze_2016_Answer_Given_options",
+    "story_cloze_2016_Choose_Story_Ending",
+    "story_cloze_2016_Movie_What_Happens_Next",
+    "story_cloze_2016_Novel_Correct_Ending",
+    "story_cloze_2016_Story_Continuation_and_Options",
+
+    # ANLI
+    *[
+        task
+        for anli_round in ["r1", "r2", "r3"]
+        for task in [
+            f"anli_GPT_3_style_{anli_round}",
+            f"anli_MNLI_crowdsource_{anli_round}",
+            f"anli_always_sometimes_never_{anli_round}",
+            f"anli_based_on_the_previous_passage_{anli_round}",
+            f"anli_can_we_infer_{anli_round}",
+            f"anli_claim_true_false_inconclusive_{anli_round}",
+            f"anli_consider_always_sometimes_never_{anli_round}",
+            f"anli_does_it_follow_that_{anli_round}",
+            f"anli_does_this_imply_{anli_round}",
+            f"anli_guaranteed_true_{anli_round}",
+            f"anli_guaranteed_possible_impossible_{anli_round}",
+            f"anli_justified_in_saying_{anli_round}",
+            f"anli_must_be_true_{anli_round}",
+            f"anli_should_assume_{anli_round}",
+            f"anli_take_the_following_as_truth_{anli_round}",
+        ]
+    ],
+
+    # CB
+    "super_glue_cb_GPT_3_style",
+    "super_glue_cb_MNLI_crowdsource",
+    "super_glue_cb_always_sometimes_never",
+    "super_glue_cb_based_on_the_previous_passage",
+    "super_glue_cb_can_we_infer",
+    "super_glue_cb_claim_true_false_inconclusive",
+    "super_glue_cb_consider_always_sometimes_never",
+    "super_glue_cb_does_it_follow_that",
+    "super_glue_cb_does_this_imply",
+    "super_glue_cb_guaranteed_true",
+    "super_glue_cb_guaranteed_possible_impossible",
+    "super_glue_cb_justified_in_saying",
+    "super_glue_cb_must_be_true",
+    "super_glue_cb_should_assume",
+    "super_glue_cb_take_the_following_as_truth",
+
+    # RTE
+    "super_glue_rte_GPT_3_style",
+    "super_glue_rte_MNLI_crowdsource",
+    "super_glue_rte_based_on_the_previous_passage",
+    "super_glue_rte_can_we_infer",
+    "super_glue_rte_does_it_follow_that",
+    "super_glue_rte_does_this_imply",
+    "super_glue_rte_guaranteed_true",
+    "super_glue_rte_justified_in_saying",
+    "super_glue_rte_must_be_true",
+    "super_glue_rte_should_assume",
+
+    # WSC
+    "super_glue_wsc.fixed_GPT_3_Style",
+    "super_glue_wsc.fixed_I_think_they_mean",
+    "super_glue_wsc.fixed_Who_or_what_is_are",
```

Comment:
I think you are missing `super_glue_wsc.fixed_Who_is_are` - that one was included in our eval and plots. Did it not show up for you or something? If so, I wonder if it got removed.

Reply:
Looks good to me, thanks.